spec/models/skin_parent_spec.rb

```diff
@@ -3,61 +3,63 @@
 describe Skin do
 
   describe "save" do
-
-    before(:each) do
-      @child_skin = Skin.new(title: "Child", css: "body {background: #fff;}")
-      @parent_skin = Skin.new(title: "Parent", css: "body {color: #000;}")
-      @child_skin.save; @parent_skin.save
-      @skin_parent = SkinParent.new(child_skin: @child_skin, parent_skin: @parent_skin, position: 1)
-    end
+    let(:child_skin) { Skin.new(title: "Child", css: "body {background: #fff;}") }
```

Comment:
Yay, thanks for changing these to `let`! 

Would using the factory and doing `build(:skin, title: "Child", css: "body {background: #fff;}")` work here?